apps/github-stack/src/github-gateway/gateway-logic/github.gateway.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  MessageBody,
+  SubscribeMessage,
+  WebSocketGateway,
+  WebSocketServer,
+} from "@nestjs/websockets";
+import { Server } from "http";
+import { EmailData } from "../../github-ineraction/domain/interface/email.interface";
+
+@WebSocketGateway({ cors: { origin: "*" } })
+export class GithubGateway {
+  @WebSocketServer()
+  server: Server;
+  @SubscribeMessage("sending-letter")
+  handleMessage(@MessageBody() email: EmailData) {
```

Comment:
Should be name sendMessage.
And on the email server side it should be called handleMessage